driver-legacy/src/main/com/mongodb/MongoClientOptions.java

```diff
@@ -242,6 +245,20 @@ public int getMaxConnecting() {
         return wrapped.getConnectionPoolSettings().getMaxConnecting();
     }
 
+    /**
+     * Returns the period of time to wait before running the first maintenance job on the connection pool.
+     * <p>
+     * Default is 0.</p>
+     *
+     * @param timeUnit unit of time to convert the value into
+     * @return The maximum number of connections a pool may be establishing concurrently.
+     * @see ConnectionPoolSettings#getMaintenanceInitialDelay
+     * @since 4.7
+     */
+    public long getMaintenanceInitialDelay(final TimeUnit timeUnit) {
```

Comment:
Let's follow existing conventions for this class:

1. Assume milliseconds rather than take a TimeUnit
2. Use int (and toIntExact) rather than long